ui-tests/src/test/java/io/syndesis/qe/CustomWebDriverProvider.java

```diff
@@ -0,0 +1,136 @@
+package io.syndesis.qe;
+
+
+import com.codeborne.selenide.WebDriverProvider;
+import lombok.extern.slf4j.Slf4j;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.firefox.FirefoxProfile;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import java.io.File;
+import java.io.FilenameFilter;
+import java.util.Hashtable;
+import java.util.Map;
+
+@Slf4j
+public class CustomWebDriverProvider implements WebDriverProvider {
+    public static final String downloadFilePath = System.getProperty("user.dir") + File.separator + "target";
```

Comment:
Hmm, I would go for something like `tmp/download` inside module dir. Also you can achieve this with `Paths.get("tmp/download").toAbsolute()`

Reply:
This property is used for integration export and all other downloads from UI tests - it should be somewhere in target dir so it is deleted with every mvn clean. What about this: 
Paths.get("target" + File.separator + "download").toAbsolutePath().toString();

Reply:
Ofc, separator works for me 👍 

I'd like to skip target dir and rather add & configure maven-clean-plugin to do it's job <module-dir>/tmp :)